cql3/cql_statement.hh

```diff
@@ -126,6 +144,10 @@ public:
     virtual seastar::shared_ptr<const metadata> get_result_metadata() const override {
         return make_empty_metadata();
     }
+
+    cql_statement_type get_statement_type() const override {
+        return cql_statement_type::NO_METADATA;
+    }
```

Comment:
`cql_statement_no_metadata` is an "abstract" class and does not correspond to any "real" operation. By implementing it here, all classes inheriting it won't be required to write their own implementation.

Reply:
I know, I mentioned this in TODO in the description. It was just a temporary solution